core/src/test/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorSpec.scala

```diff
@@ -129,7 +129,13 @@ class SubmissionMonitorSpec(_system: ActorSystem)
     assertResult(
       ignoreStatusLastChangedDate(
         ExecutionServiceStatusResponse(
-          workflowsRecs.map(workflowRec => scala.util.Success(None))
+          workflowsRecs.map { workflowRec =>
+            scala.util.Success(
+              Option(
+                (workflowRec.copy(cost = Option(BigDecimal.valueOf(0))), None)
+              )
+            )
+          }
```

Comment:
previously, the monitor would return None because the workflow's status hadn't changed. Now, since the monitor compares both status and cost, and the cost _has_ changed, it returns Some instead of None.